osgar/turn.py

```diff
@@ -30,9 +30,10 @@ def update(self):
             print(self.time, 'Go', channel)
         if channel == 'pose2d':
             x, y, heading = self.pose2d
-            pose = (x/1000.0, y/1000.0, math.radians(heading/100.0))
+            pose = (x/1000.0, y/1000.0, math.radians(heading/100.0))            
```

Comment:
You are just adding white space to the end of the line here.

Reply:
fixed